tests/Slovakia/EasterMondayTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/**
+ *  This file is part of the Yasumi package.
+ *
+ *  Copyright (c) 2015 - 2016 AzuyaLabs
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ *
+ *  @author Sacha Telgenhof <stelgenhof@gmail.com>
+ */
+
+
+namespace Yasumi\tests\Slovakia;
+
+use DateTime,
+	DateTimeZone,
+	Yasumi\Holiday,
+	Yasumi\tests\YasumiTestCaseInterface;
+
+
+/**
+ * Class for testing a holiday in Slovakia.
+ *
+ *
+ * @package Yasumi\tests\Slovakia
+ * @author  Andrej Rypak (dakujem) <xrypak@gmail.com>
+ */
+class EasterMondayTest extends SlovakiaBaseTestCase implements YasumiTestCaseInterface
+{
+	/**
+	 * The name of the holiday
+	 */
+	const HOLIDAY = 'easterMonday';
+
+
+	/**
+	 * Tests the holiday defined in this test.
+	 *
+	 * @dataProvider HolidayDataProvider
+	 *
+	 * @param int      $year     the year for which Christmas Day needs to be tested
+	 * @param DateTime $expected the expected date
+	 */
+	public function testHoliday($year, $expected)
+	{
+		$this->assertHoliday(self::REGION, self::HOLIDAY, $year, $expected);
+	}
+
+
+	/**
+	 * Returns a list of test dates used for assertion of the holiday defined in this test
+	 *
+	 * @return array list of test dates for the holiday defined in this test
+	 */
+	public function HolidayDataProvider()
```

Comment:
Do you think you can use a date generator rather than a fixed list of test dates? That would be much more sustainable.